lib/cc/services/github_pull_requests.rb

```diff
@@ -73,4 +89,48 @@ def response_includes_repo_scope?(response)
   def test_status_code
     422
   end
+
+  def should_post_welcome_comment?
+    config.welcome_comment_enabled && @payload.fetch("first_contribution", false)
+  end
+
+  def welcome_comment_markdown_header
+    <<-HEADER
+Hey, @#{@payload.fetch("author_username")}-- Since this is the first PR we've seen from you,
+here's some things you should know about contributing to #{github_slug}:
+
+    HEADER
+  end
+
+  def admin_only_footer
+    <<-FOOTER
+
+* * *
+
+Quick note: By default, Code Climate will post the above comment on the *first* PR it sees from each contributor.
+If you'd like to customize this message or disable this, go [here](#{@payload.fetch("pull_request_integration_edit_url")}).
+    FOOTER
+  end
+
+  def welcome_comment_markdown
+    header = welcome_comment_markdown_header
+    body = config.welcome_comment_markdown
+    if @payload.fetch("author_is_site_admin")
+      header + body + admin_only_footer
+    else
+      header + body
+    end
+  end
+
+  def post_welcome_comment
+    pull_request_number = @payload.fetch("number")
+    comments_url = "#{config.base_url}/repos/#{github_slug}/issues/#{pull_request_number}/comments"
+
+    params = { body: welcome_comment_markdown }
+
+    # FIXME: is this appropriate? when we receive the pending event, we used to just update the status for the commit,
```

Comment:
OK my new plan is to merge the two network responses into one service event that reflects the success of both. It will be `ok: true` when both requests succeeded, and it will have the [metadata](https://github.com/codeclimate/codeclimate-services/blob/e0e48d49bb3f268ee07d094b1101799ad2247bee/lib/cc/service/http.rb#L93-L100) from both responses for debugging purposes